client/src/components/ManageBooking/BookingList/BookingList.tsx

```diff
@@ -0,0 +1,64 @@
+import { Grid, Avatar, Box } from '@material-ui/core';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import SettingsIcon from '@material-ui/icons/Settings';
+import useStyles from './useStyles';
+import { useAuth } from '../../../context/useAuthContext';
+import BookingGrid from './BookingGrid';
+
+const BookingList = (): JSX.Element => {
+  const classes = useStyles();
+  const { loggedInUser } = useAuth();
+  console.log(loggedInUser);
```

Comment:
We should remove this before merging as well.

Reply:
removed